utility/src/main/java/org/oppia/android/util/parser/OppiaImage.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.util.parser
+
+sealed class OppiaImage {
+  class BitmapImage(val urlDrawable: UrlImageParser.UrlDrawable) : OppiaImage()
+  class SvgImage(val urlDrawable: UrlImageParser.UrlDrawable) : OppiaImage()
```

Comment:
We shouldn't be referencing ``UrlImageParser`` things here--that's a dependency inversion. ``OppiaImage`` is part of image loading, and ``UrlImageParser`` depends on image loading.

Reply:
Done